fbfmaproom/fbftable.py

```diff
@@ -0,0 +1,80 @@
+import os
+import time
+import io
+import datetime
+import random
+import urllib.parse
+import numpy as np
+import pandas as pd
+from pathlib import Path
+import xarray as xr
+import flask
+from dash import Dash, dcc, html
+from dash.dependencies import Output, Input, State, ALL
+from dash.exceptions import PreventUpdate
+import __about__ as about
+import pyaconf
+import dash_bootstrap_components as dbc
+import uuid
+from collections import OrderedDict
```

Comment:
UUID I used for generating IDs for the target of tooltips. Bootstrap components I use for tooltips. I added the collections import "just in case" but it's probably unnecessary. Pyaconf is indeed not used.

Reply:
Realized you meant the imports in general